templates/tfengine/components/project/project/main.tf

```diff
@@ -1,28 +1,47 @@
-# Copyright 2020 Google Inc.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#      http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
+{{- /* Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */ -}}
 
 # Create the project, enable APIs, and create the deletion lien, if specified.
 module "project" {
   source  = "terraform-google-modules/project-factory/google"
-  version = "~> 7.0"
+  version = "~> 8.0.0"
 
-  name                    = var.name
+  name                    = "{{.PROJECT_ID}}"
   org_id                  = var.org_id
   folder_id               = var.folder_id
   billing_account         = var.billing_account
-  lien                    = var.enable_lien
-  activate_apis           = var.apis
+  lien                    = {{get . "ENABLE_LIEN" "true"}}
```

Comment:
Should this be in quotes? It seems like a bool.

Reply:
While it doesn't actually make a diff because even if we return a string by default it wouldn't be surrounded by quotes, but I agree it's clearer to not put quotes so I removed it.